source/css/sass/_normalize.scss

```diff
@@ -51,47 +51,64 @@ h6 {
   @include h6-spacing();
 }
 
-p {
-  @include body-para();
+address,
+article,
+aside,
+blockquote,
+caption,
+details,
+dl,
+fieldset,
+figcaption,
+figure,
+footer,
+form,
+header,
+hr,
+main,
+nav,
+ol,
+p,
+pre,
+section,
+table,
+ul {
+  @include block-spacing();
+}
+
+figure {
+  @include margin(0, "inline");
+}
+
+dl,
+ol,
+ul {
+  dd > &,
+  li > & {
+    @include block-spacing(0);
+  }
 }
 
 a {
   color: $color-primary-normal;
   text-decoration: none;
 }
 
-ul,
-ol {
-  @include blg-spacing("block-end", "small", "margin");
-  margin-top: 0;
-}
-
-dl {
-  @include blg-spacing("block-end", "small", "margin");
-  margin-top: 0;
-}
-
-li,
-dt,
-dd {
-  @include body-typography();
-}
-
 dt {
   font-weight: bold;
 
   dd + & {
-    @include blg-spacing("block-start", "small", "margin");
+    @include block-spacing($start: $baselinegrid-space-small, $end: 0);
```

Comment:
Might be able to avoid this if we reversed it all? (As things know what they follow, not what they precede.)